### PR TITLE
Readme: update screenshots 

### PR DESCRIPTION
**Pull request by illusionyy**

### CODE REVIEW

**Comment by illusionyy** [comment]
Preview https://github.com/illusion0001/GoldHEN_Cheat_Manager/tree/img#readme

**Comment by bucanero** [comment]
can you convert the images back to .jpg? it's just a Readme and we don't need pixel perfect images, so I rather have small-sized images that load faster.

For example the original main-menu screenshot .jpg is about 110 Kbytes, and the new .png is over 1 Mbyte (like 10x bigger)

**Comment by illusionyy** [comment]
Done 👍

File: README.md [hunk 1]
```diff
@@ -38,7 +47,7 @@ It will open the context menu on the screen. Press ![O button](https://github.co
 
 # Online Database
 
-The application also provides direct access to the [Online Cheat database](https://github.com/GoldHEN/GoldHEN_Cheat_Repository), the official GoldHEN repository of cheat files for PlayStation 4 games.
+The application also provides direct access to the Online [Cheat](https://github.com/GoldHEN/GoldHEN_Cheat_Repository) and [Patch](https://github.com/GoldHEN/GoldHEN_Patch_Repository) database, the official GoldHEN repository of cheat and custom game patch files for PlayStation 4 games.
```
Comment (bucanero) [inline README.md]:
```suggestion
The application also provides direct access to the Online [Cheat](https://github.com/GoldHEN/GoldHEN_Cheat_Repository) and [Patch](https://github.com/GoldHEN/GoldHEN_Patch_Repository) databases, the official GoldHEN repositories of cheats and custom game patches for PlayStation 4 games.
```

**Review by bucanero — APPROVED** [review]

**Pull request merged**